src/utils/config.ts

```diff
@@ -486,6 +486,33 @@ export async function getProjectRoot() {
   return root
 }
 
+export async function getDirectoryContainingLintConfig() {
+  let lintConfigFolder = ''
```

Comment:
I think it might be best if I exposed this function from the lint library, since the logic is all there already.

Reply:
now using `sasjs/lint/utils/getProjectRoot`